idlgen/tests/examples/structs.rs

```diff
@@ -0,0 +1,8 @@
+struct Point {
+    x: f64,
+    y: f64,
+}
+struct ChessSquare {
```

Comment:
Before each struct there should be #[derive(serde::Deserialize, serde::Serialize, dust_dds::DdsType, dust_dds::DdsSerde)]